src/ps2/card_emu/ps2_sd2psxman.c

```diff
@@ -0,0 +1,78 @@
+#include <stdio.h>
+#include "ps2/ps2_cardman.h"
+#include "ps2/card_emu/ps2_memory_card.h"
+#include "ps2/card_emu/ps2_sd2psxman.h"
+#include "gui.h"
+
+#include "pico/time.h"
+#include "ps2/card_emu/ps2_sd2psxman_commands.h"
+
+volatile uint8_t sd2psxman_cmd;
+volatile uint8_t sd2psxman_mode;
+volatile uint16_t sd2psxman_cnum;
+char sd2psxman_gameid[251] = {'S', 'L', 'U', 'S', '-', '2', '0', '5', '5', '4', '\0'};
```

Comment:
Can you please default to { '\0' } for a clean init?

Reply:
Sure, I was using that value for testing and forgot to remove it.